Gallery/Contours/NCL_coneff_11.py

```diff
@@ -0,0 +1,127 @@
+"""
+NCL_coneff_11.py
+================
+
+This script illustrates the following concepts:
+   - Filling contours with multiple shaded patterns
+   - Filling contours with stippling (solid dots)
+   - Changing the size of the dot fill pattern in a contour plot
+   - Overlaying a stipple pattern to show area of interest
+   - Changing the density of contour shaded patterns
```

Comment:
I recommend updating the language here to be consistent with Matplotlib (i.e. "shaded patterns" --> "hatches") and to mention the work arounds we had to find (i.e. creating a color bar with colored hatch/stippling patterns). This will make it easier for NCL users converting to Python to learn the new terminology and for all users to find examples they're searching for.